tests/kani/StdLink/overlapping_traits.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright Kani Contributors
```

Comment:
Since this example comes from a book, shouldn't we use the copyright for modifications?

Reply:
Good point